modules/update_helper_checklist/src/UpdateChecklist.php

```diff
@@ -250,8 +250,9 @@ protected function checkAllListPoints($status = TRUE) {
       }
     }
 
+    $all_items = $update_check_list->get(ChecklistapiChecklist::PROGRESS_CONFIG_KEY . ".#items");
```

Comment:
There is a similar count() on the items on line 205 of this file. We should investigate if we need this fix there as well

Reply:
Plus, we should take this out of the PR and do a followup, this seems not to be related to the profile support.

Reply:
👍